book/html.md

```diff
@@ -417,7 +419,7 @@ def add_tag(self, tag):
         parent.children.append(node)
 ```
 
-This code looks right, but if you test it out it won't seem to help.
+This code looks right, but it doesn't quite work right.
```

Comment:
This edit on its own isn't important, but it's to avoid a bad line break in the middle of the next line (see page 82 in the proof)